Java Migration/Azure_DevOps_Server_2019.md

```diff
@@ -0,0 +1,41 @@
+
+
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for Azure DevOps Server 2019.
+
+![Java Migration flow](https://github.com/msftazdev/Code-Search/blob/msftazdev-patch-1/Java%20Migration/flow1.png)
+
+## Step 1: Pause Search indexing
+Go to https://github.com/Microsoft/Code-Search and find the right folder based on the TFS version you are using. For Azure DevOps Server 2019, go to https://github.com/Microsoft/Code-Search/tree/master/Azure_DevOps_Server_2019. To pause all indexing, execute the script PauseSearchIndexing.ps1 from Windows PowerShell with administrative privileges. You will be prompted to enter:
+
+* The SQL server instance name where the TFS configuration database resides.
+* The name of the TFS configuration database.
+
+## Step 2: Stop Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-2.4.1\bin"
+### Stop the service:
+execute "elasticsearch-service.bat stop"
+
+## Step 3: Download and Install Azul Zulu Java 
+Download and install [OpenJDK 8u201](https://cdn.azul.com/zulu/bin/zulu8.34.0.1-ca-jdk8.0.201-win_x64.msi)
```

Comment:
Does it need to be the particular patch number or any 8 version?

Reply:
Makes sense, but aren't updates released every month? Pinning it to a specific version means that you'll need to update this regularly